core/felt/felt.go

```diff
@@ -11,12 +11,12 @@ import (
 )
 
 type Felt struct {
-	val fp.Element
+	Val fp.Element
```

Comment:
Use `Impl()` instead of making this field public

Reply:
Thanks